### ISSUE TITLE
Add package to carthage and Cocoa Pods

### ISSUE THREAD

**Issue by sacOO7** [sacOO7/socketcluster-client-swift#1]
Publish library on carthage and Cocoa Pods for better dependency management across various platforms.

**Comment by Punith1234** [comment]
@sacOO7 - Any updates on this it would be great if you could add Cocoa Pods package

**Comment by sacOO7** [comment]
Hi @Punith1234 I will be working on it. Till then can you use swift package manager to handle dependency.

**Comment by Punith1234** [comment]
Hi @sacOO7  - We've used cocoa pods in our project and i'm finding very difficult to integrate it through Swift manager - Is there any way to which you know so that it can help us

**Comment by sacOO7** [comment]
Yes, but I think it will take a little time. Can you do one thing instead? Add all the source files in the project and install its dependencies via CacoaPods.

**Comment by Punith1234** [comment]
Hi @sacOO7  did you get any chance to update on this 
Currently we're using https://github.com/abpopov/SocketCluster-ios-client in our Swift project 
And it as lot of issues with persistence of subscription. It'll be very help full for us if you could update this

**Comment by sacOO7** [comment]
Hi @Punith1234 , I have uploaded library on cocoapod, it will be available for use very soon..Till then you can use it as a GitHub link pod dependency.

**Comment by sacOO7** [comment]
Hi @tech4solitude , any updates on this? I will close the issue if your problem is resolved

**Comment by sacOO7** [comment]
Cocoapods achieved, need to upload it on **Carthage**.
